core/domain/email_manager.py

```diff
@@ -209,7 +209,7 @@ def log_new_error(*args, **kwargs):
     []
 )
 
-NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_EMAIL_INFO = {
+NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_EMAIL_DATA = {
```

Comment:
REVIEWER_NOTIFICATION_EMAIL_DATA

(try not to start constant names with verbs)

Reply:
I think it'd be better to have CONTRIBUTOR_DASHBOARD as a prefix so as to not confuse it with the creator dashboard but ya I made the change